momentGW/pbc/tda.py

```diff
@@ -34,6 +36,22 @@ class dTDA(MoldTDA):
         Default value is `None`.
     """
 
+    def __init__(
+        self,
+        gw,
+        nmom_max,
+        integrals,
+        fc=False,
+        mo_energy=None,
+        mo_occ=None,
+    ):
+        super().__init__(gw, nmom_max, integrals, mo_energy=mo_energy, mo_occ=mo_occ)
+        self.fc = fc
+        if self.fc:
+            q = np.array([1e-3, 0, 0]).reshape(1, 3)
+            self.q_abs = self.kpts.cell.get_abs_kpts(q)
+            self.qij = self.build_pert_term(self.q_abs[0])
```

Comment:
`q_abs` is O(1) to compute right? If so, can we make it a property instead.

I'd prefer not storing `qij` in the class, but if it's needed in several places then maybe it's for the best. Maybe a more descriptive variable name?

Reply:
Changed the variable name, still not great but not sure what makes more sense.